pkgs/development/cuda-modules/generic-builders/multiplex.nix

```diff
@@ -20,7 +20,7 @@
   # The featureRelease is used to populate meta.platforms (by way of looking at the attribute names)
   # and to determine the outputs of the package.
   # shimFn :: {package, redistArch} -> AttrSet
```

Comment:
No longer true?

Reply:
Still true -- we still use the attributes of the release to see which platforms are supported.

The type signature of `shimsFn` is still the same, I just wanted to throw if a `shimsFn` wasn't provided rather than waiting until trying to call `shimsFn` to find out that it wasn't provided.